tests/log-data-per-sec-interval-10/README.md

```diff
@@ -0,0 +1,4 @@
+# Test Description
+
+Test if the timed counters from the counters.c are calculated correctly in stats.log, 
+applying a stats interval of 10 secoonds
```

Comment:
Could you also add a section indicating where the pcap comes from, and another for the redmine ticket? :)

Reply:
For the descriptions in these tests, I think it's better if we use something more high-level.
So, something related to the pkts counters that are being tested, I guess?

Reply:
Hi, 

I have added a random pcap file that is not special, so is it better use a file that already exists in the project?

Reply:
I'll think of a better description following your advice, thanks!